src/resources/postcss/tickets-tables.pcss

```
@@ -136,20 +137,33 @@
 		padding: 0;
 
 		&:hover {
-			color: #00a0d2;
+			color: #727272;
 		}
 
 		.ticket_edit_text,
-		.global_capacity_edit_text {
+		.global_capacity_edit_text,
+		.ticket_delete_text {
 			font-size: 0;
 
 			&::before {
 				content: '\f464';
 				font-family: 'dashicons';
-				font-size: x-large;
+				font-size: 20px;
 				vertical-align: top;
 			}
 		}
+
+		.ticket_delete_text {
+			font-size: 0;
+
+			&::before {
```

for `:before` and `:after` we use single-colon notation. when the linter is in place, this will be flagged.

```suggestion
			&:before {
```

---

still missing this one

---

might as well combine it with the `.ticket_delete_text` above.